_pytest/tmpdir.py

```diff
@@ -54,9 +54,8 @@ def getbasetemp(self):
             else:
                 # use a sub-directory in the temproot to speed-up
                 # make_numbered_dir() call
-                import getpass
                 temproot = py.path.local.get_temproot()
-                rootdir = temproot.join('pytest-of-%s' % getpass.getuser())
+                rootdir = temproot.join('pytest-of-%s' % get_user())
```

Comment:
sorry i was unclear.  I meant not creating a subdir if we don't have a username. 

Suppose two users on a Windows machine where each executes tox which executes pytest.  Will they not conflict on creating pytest-tox at the temproot by default?